vocab/templates/vocab.html

```diff
@@ -77,27 +77,44 @@ <h1>Words</h1>
                   <label for="attempt">Word:</label>
                   <input id="attempt" name="attempt" type="text" width="15"
                      autocomplete="off"  autofocus />
-                  <input type="submit" name="try" />
                </form>
             </div>
          </div>
+         <p id="response"> </p>
+         <!-- JavaScript will insert text here -->
+         <script>
+            //// Suppress normal form submission
+            $("#entry").submit( function(event) {
+               event.preventDefault();
+              });
+            
+            $("#attempt").keyup(function(event) {
+               var txt = $("#attempt").val();  // Current content of the input field
+               var keycode = event.which;      // They key that just went up
+               var letter = String.fromCharCode(keycode);
+            
+               if (! "ABCDEFGHIJKLMNOPQRSTUVWXYZ".includes(letter)) { return }
+
+            $.getJSON( "/_check", { text: txt },
+               function(data) {
+               message = data.result.message;
+               found = data.result.found;
+               success = data.result.success;
+               
+               if (success){
+                  window.location.replace("success.html")}
```

Comment:
this should be window.location.replace("/success")
otherwise, it redirects to a 404 error page instead of the success page

```suggestion
                  window.location.replace("/success")
```